### DISCUSSION TITLE
Case-insensitive string constraint

### DISCUSSION THREAD

**Discussion by leoluk** [cue-lang/cue#1638] [Q&A] [answered]
How can I do a case-insensitive string match constraint?

Neither regex matches nor `strings.Contains` seem to allow for case-insensitive matching.

**Comment by jornh** [comment, marked as answer]
Looks like a `(?i)` prefix for case insensitive like in a Go regex is your friend.

https://cuelang.org/play/?id=i-locxN-CTa#cue@export@cue

**Reply by mvdan** [reply]
Marking as the answer :) The CUE spec also links to https://github.com/google/re2/wiki/Syntax.
